config.cfg

```diff
@@ -206,7 +206,7 @@ cloud_providers:
     image: Ubuntu 22.04 Jammy Jellyfish
     arch: x86_64
   hetzner:
-    server_type: cx22
+    server_type: cpx21
```

Comment:
We should probably use CPX11 instead of CPX21.

Reply:
OK. I'm not sure what the resource requirements are, and they don't map neatly between Intel and AMD types, so I guessed the cpx21 was the closest to the cx22.